plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/messages.properties

```diff
@@ -5,6 +5,7 @@ CloudSdkPreferencePage_3=URL malformed
 CloudSdkPreferencePage_4=Cannot launch a browser
 CloudSdkPreferencePage_5=&SDK location:
 CloudSdkPreferencePage_6=No SDK found: {0}
+AppEngineJavaComponentsNotInstalled=App Engine Java components not installed
```

Comment:
We should give some help on how to add the java components ('_please run `gcloud install app-engine-java`_ from the command line_'?)